version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.8-AV4"
+version in ThisBuild := "0.0.9-AV4-test"
```

Comment:
Would you be so kind to also update `.gitignore` for IntelliJ usage? It's pretty annoying rn.